audio/exports/exportmidi.cpp

```diff
@@ -90,7 +90,7 @@ void ExportMidi::writeHeader()
             Staff* staff  = cs->staff(staffIdx);
 
             QByteArray partName = staff->partName().toUtf8();
-            int len = partName.length() + 1;
+            int len = partName.length();
```

Comment:
This line was correct. An increasing of the length must be here.

See Qt's documentation: https://doc.qt.io/qt-5/qbytearray.html#data

Reply:
Since we're doing a `memcpy` instead of a `strcpy`, I don't think the `+ 1` is necessary.